app/activeplan/homepage/get.controller.js

```diff
@@ -1,13 +1,21 @@
 const { logger } = require('../../../common/logging/logger')
+const { getSentencePlanComments } = require('../../../common/data/sentencePlanningApi')
+const { getCommentText } = require('../../../common/utils/getCommentText')
 
 const getHomepage = async (req, res) => {
   const {
-    session: { planStarted = false },
+    session: { planStarted = false, 'x-auth-token': token },
     params: { planId, id },
   } = req
   try {
     delete req.session.planStarted
-    return res.render(`${__dirname}/index`, { planId, id, planStarted })
+    const renderDetails = {}
+
+    // get contact arrangements
+    const comments = await getSentencePlanComments(planId, token)
+    renderDetails.contactArrangements = getCommentText(comments, 'LIAISON_ARRANGEMENTS')
```

Comment:
'renderDetails' serves no purpose from what I can see; it temporarily nests contactArrangements before being destructed.
`contactArrangements = getCommentText(comments, 'LIAISON_ARRANGEMENTS')`
...and then drop it straight into the return object:
return res.render(`${__dirname}/index`, { planId, id, planStarted, contactArrangements })

Reply:
Ok, read the comment on the other MR which supersedes this 😬👍